OpenOversight/app/main/model_view.py

```diff
@@ -225,8 +216,15 @@ def get_department_id(self, obj):
 
     def populate_obj(self, form, obj):
         form.populate_obj(obj)
-        if hasattr(obj, "updated_at"):
-            obj.updated_at = datetime.datetime.now()
+
+        # if the object doesn't have a creator id set it to current user
+        if hasattr(obj, "created_by") and not getattr(obj, "created_by"):
+            obj.created_by = current_user.get_id()
+        # if the object keeps track of who updated it last, set to current user
+        if hasattr(obj, "last_updated_at"):
+            obj.last_updated_at = datetime.datetime.now()
+            obj.last_updated_by = current_user.get_id()
```

Comment:
Does this trigger on edits or just new objects?

Reply:
This code only triggers on new objects but I've also updated `populate_obj` which updates the last_updated on edits